packages/manager/apps/pci-workflow/src/pages/onboarding/OnBoarding.page.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import {
+  ODS_TEXT_LEVEL,
+  OdsBreadcrumbAttributeItem,
+} from '@ovhcloud/ods-components';
+import { OsdsBreadcrumb, OsdsText } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { Outlet, useNavigate, useParams } from 'react-router-dom';
+import {
+  isDiscoveryProject,
+  OnboardingLayout,
+  PciDiscoveryBanner,
+  RedirectionGuard,
+  useProject,
+  useProjectUrl,
+} from '@ovhcloud/manager-components';
+import { Suspense } from 'react';
+import { useWorkflows } from '@/api/hooks/workflows';
+
+export default function OnBoardingPage() {
+  const { t } = useTranslation();
+  const { t: tOnBoarding } = useTranslation('onboarding');
+  const { projectId } = useParams();
+  const { data: project } = useProject(projectId);
+  const urlProject = useProjectUrl('public-cloud');
+  const navigate = useNavigate();
+  const { data: workflows, isPending } = useWorkflows(projectId);
+
+  const breadcrumbItems: OdsBreadcrumbAttributeItem[] = [
+    {
+      href: urlProject,
+      label: project?.description,
+    },
+    {
+      label: t('pci_workflow_title'),
+    },
+  ];
+
+  return (
+    <RedirectionGuard
+      isLoading={isPending}
+      route={`/pci/projects/${projectId}/workflow`}
+      condition={workflows?.length > 0}
+    >
+      {project && <OsdsBreadcrumb items={breadcrumbItems} />}
```

Comment:
no need for this conditional rendering while you use optional chaining in here label: project?.description